### PR TITLE
Updated organization users add/remove commands to REST

### PR DESCRIPTION
**Pull request by lukasbals**
## Summary of the changes / Why this is an improvement

The `organizations users add|remove` commands now require the arguments `--user` and `--org-id`. Additionally, the `organizations users add` command requires the `--role` argument.

## Checklist

 - [x] [CLA](https://crate.io/community/contribute/cla/) is signed

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -5,6 +5,12 @@ Changes for croud
 Unreleased
 ==========
 
+- [Breaking] Updated ``organizations users add/remove`` commands from GraphQL to REST.
```
Comment (chaudum) [inline CHANGES.rst]: From a user's point of view the API endpoint is irrelevant, all it matters is that the arguments changed.

Comment (chaudum) [reply CHANGES.rst]: BREAKING: The `organizations user add|remove` commands now require the arguments ...

File: CHANGES.rst [hunk 2]
```diff
@@ -5,6 +5,10 @@ Changes for croud
 Unreleased
 ==========
 
+- [Breaking] The ``organizations user add|remove`` commands now require the arguments
+  ``--user`` and ``--org-id`` additionally the add command requires the ``--role``
```
Comment (chaudum) [inline CHANGES.rst]:
```suggestion
  ``--user`` and ``--org-id``. Additionally the ``organizations users add`` command requires the ``--role``
```

File: CHANGES.rst [hunk 3]
```diff
@@ -5,6 +5,10 @@ Changes for croud
 Unreleased
 ==========
 
+- [Breaking] The ``organizations user add|remove`` commands now require the arguments
```
Comment (chaudum) [inline CHANGES.rst]:
```suggestion
- [Breaking] The ``organizations users add|remove`` commands now require the arguments
```

File: CHANGES.rst [hunk 4]
```diff
@@ -5,6 +5,10 @@ Changes for croud
...
+  ``--user`` and ``--org-id`` additionally the add command requires the ``--role``
+  argument. Before, both commands only required the ``--user`` argument.
```
Comment (chaudum) [inline CHANGES.rst]:
```suggestion
  argument.
```

File: tests/unit_tests/test_commands.py [hunk 5]
```diff
@@ -338,131 +338,55 @@ def test_list(self, mock_send, mock_run, mock_load_config):
...
+            user,
             "--org-id",
             org_id,
         ]
-        self.assertGql(mock_run, argv, expected_body, expected_vars)
+        self.assertRest(
+            mock_send,
+            argv,
+            RequestMethod.DELETE,
+            f"/api/v2/organizations/{org_id}/users/{user}/",
```
[the remark this answers is not in the document]
Comment (MarkusH) [reply tests/unit_tests/test_commands.py]: Yikes, good point. We should do that for all URLs we construct in croud.

[the remark this answers is not in the document]
Comment (chaudum) [reply tests/unit_tests/test_commands.py]:
```py
from yarl import URL
url = URL(self.url).with_path(endpoint)
```

File: croud/session.py [hunk 6]
```diff
@@ -77,7 +78,7 @@ def _get_conn(self) -> TCPConnector:
         body: dict = None,
         params: dict = None,
     ) -> ClientResponse:
-        url = self.url + endpoint
+        url = URL(self.url).with_path(endpoint)
```
Comment (chaudum) [inline croud/session.py]: Please add a test :)

File: tests/unit_tests/test_session.py [hunk 7]
```diff
@@ -71,6 +72,9 @@ def test_query_success(self, mock_token, mock_env, variables):
                         "/graphql",
                         {"query": me_query, "variables": variables},
                     )
+                    assert resp.url == URL("https://cratedb.local").with_path(
```
Comment (chaudum) [inline tests/unit_tests/test_session.py]: You could pass the full url string directly

```suggestion
                    assert resp.url == URL("https://cratedb.local/graphql")
```

**Pull request merged**